src/libertem/web/server.py

```diff
@@ -119,9 +119,12 @@ def sig_exit(signum, frame, shared_state):
     )
 
 
-def main(host, port, event_registry, shared_state):
+def main(host, port, log_level, event_registry, shared_state):
+    numeric_level = getattr(logging, log_level.upper(), None)
+    if not isinstance(numeric_level, int):
+        raise ValueError(f'Invalid log level: {log_level}')
```

Comment:
I would prefer to have this validation logic in the CLI part instead. Then, you can [raise a `click.UsageError` instead](https://github.com/LiberTEM/LiberTEM/blob/b7c1b148727eb5bb89ee00a16b35be3901694d87/scripts/release#L545), which should give a better user experience than just printing a Traceback.